status-im/status-react
# feat: implement audio message component

fixes: #13530 #15536 

### Summary

Audio message component implementation

#### Screenshots

<img src='https://user-images.githubusercontent.com/18485527/234355286-b787e5b7-51d5-4f9a-897b-ddf151808d24.png' width='300'> <img src='https://user-images.githubusercontent.com/18485527/234355306-4c5c80c7-6afa-41f9-a1c8-0c21d8dc56a3.png' width='300'>

### Testing notes

In #13530, these items were listed to be resolved. Those not checked should be worked separately in follow up issues / PRs.
- [x] Check the [play] button is not frozen after coming back from background or sleep mode
- [x] Check audio is able to be played from the pinned messages
- [x] Check banner UI for pinned audio

##### Should be implemented separately:
- Check after the call incoming the [play] button should turn from 'play' to 'pause'
- Check the max seconds for audio = 2:00 min (worked here #15543)
- Check audio into 'Jump to' page
- Check audio into Activity Center in reply
- Check audio into push notification

#### Platforms

- Android
- iOS

#### Areas that maybe impacted

##### Functional

- 1-1 chats
- public chats
- group chats

### Steps to test

- Open Status
- Open a chat with audio messages
- Or send audio
- Verify correct UI and behavior of the audio message component

status: ready
